docs/platform/concepts/service_backups.rst

```diff
@@ -139,12 +143,12 @@ You can control the persistence feature using ``redis_persistence`` under **Adva
 
 .. note::
 
-    AOF persistence is currently not supported by the Aiven for the managed Redis service.
+    AOF persistence is currently not supported by Aiven for the managed Redis service.
 
 Aiven for InfluxDB®
 '''''''''''''''''''
 
-Aiven for InfluxDB backups are taken every 12 hours with 2.5 days of retention. InfluxDB® is automatically backed up, encrypted, and uploaded to the Aiven's S3 account in the same region. When an instance has to be rebuilt, the backup is downloaded and restored to create a new instance.
+Aiven for InfluxDB backups are taken every 12 hours with 2.5 days of retention. InfluxDB® is automatically backed up, encrypted, and uploaded to Aiven's S3 account in the same region. When an instance has to be rebuilt, the backup is downloaded and restored to create a new instance.
```

Comment:
I'd say we need SOME article for "Aiven's S3 account", depending on whether we're talking about a generic concept here (the) or any individual account (an). Again, no good answer here - up to you.

Reply:
I wouldn't worry too much about InfluxDB, coz this might soon be removed from Aiven's managed services.